lib/active_admin/batch_actions/controller.rb

```diff
@@ -5,9 +5,10 @@ module Controller
       # Controller action that is called when submitting the batch action form
       def batch_action
         if action_present?
-          selection =            params[:collection_selection] ||  []
-          inputs    = JSON.parse params[:batch_action_inputs]  || '{}'
-          inputs    = inputs.with_indifferent_access.slice *instance_exec(&current_batch_action.inputs).keys
+          selection  =            params[:collection_selection] ||  []
+          inputs     = JSON.parse params[:batch_action_inputs]  || '{}'
```

Comment:
I know there is a minimal performance difference between single and double quotes. But I prefer option B from the https://github.com/bbatsov/ruby-style-guide#consistent-string-literals, while this style can used for each case, with single quotes some times you need to switch to double quotes